Release-Checklist.md

```diff
@@ -0,0 +1,79 @@
+Change release process, instead create a release candidate we can test the HEAD what will be the last commit using new automated obs package generation we can generate and test commits based on HEAD.
+
+- [ ] Raise PRs to update the `VERSION` file in the following repositories:
+   - [ ] [agent][agent].
+   - [ ] [proxy][proxy].
+   - [ ] [runtime][runtime].
+   - [ ] [shim][shim].
+   - [ ] [throttler][throttler].
+
+  Note that the "phase" element of the project encoded in the version strings needs to match for all components. For example, when a `beta` is released, the version string for *all* components should show `beta`.
+
+- [ ] Ensure all CI tests pass **for all architectures**.
+
+- [ ] Get confirmation from metrics CI that performance is within acceptable limits.
+
+- [ ] Create an **annotated tag** for the new release version for the following repositories:
+   - [ ] [agent][agent].
+   - [ ] [proxy][proxy].
+   - [ ] [runtime][runtime].
+   - [ ] [shim][shim].
+   - [ ] [throttler][throttler].
+
+   This is required by `git describe`.
+
+- [ ] Generate OBS packages based on `HEAD`:
+   - [ ] [agent][agent].
+   - [ ] [guest kernel][kernel].
+   - [ ] [image][image].
+   - [ ] [initrd][initrd].
+   - [ ] [proxy][proxy].
+   - [ ] [qemu-lite][qemu-lite].
+   - [ ] [runtime][runtime].
+   - [ ] [shim][shim].
+   - [ ] [throttler][throttler].
+
+- [ ] Installation tests (must be done for major releases)
+  - [ ] Centos.
+  - [ ] Fedora.
+  - [ ] Ubuntu.
+
+- [ ] Check if any of the following need to be updated:
+ - [ ] Architecture document.
+ - [ ] [Developer guide](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md).
+ - [ ] Installation documentation.
+ - [ ] [Limitations document](https://github.com/kata-containers/documentation/blob/master/Limitations.md).
+
+
+- [ ] Write release notes:
+  - [ ] Link to Limitations document.
+  - [ ] Brief summary of known issues (with links to appropriate Issues/PRs) for any late-breaking issues.
+  - [ ] List new features.
+  - [ ] List resolved bugs and limitations.
+  - [ ] Version of Docker (ideally range of versions, or "up to version X") supported by the release.
+  - [ ] CRI-O version.
+  - [ ] `cri-containerd` version.
+  - [ ] Version of the OCI spec (ideally range of versions, or "up to version X") supported by the release.
+  - [ ] Version of image used by the release (guest kernel version, guest O/S version and agent version).
+  - [ ] Add links to Installation instructions.
+  - [ ] Document any common vulnerabilities and exposures (CVEs) fixed with links to the CVE database.
+
+- [ ] Upload any assets to github:
```

Comment:
Do we want to upload the kernel and images for the release or only be provided as part of the packages?

Reply:
Yeah, I wasn't sure about this item. I'll remove it for now...

/cc @grahamwhaley, @erick0z, @egernst.